tests/pubspec.yaml

```diff
@@ -8,12 +8,6 @@ environment:
   sdk: ">=2.16.0 <3.0.0"
 
 dependencies:
-  cloud_firestore:
-    path: ../packages/cloud_firestore/cloud_firestore
-  cloud_firestore_platform_interface:
-    path: ../packages/cloud_firestore/cloud_firestore_platform_interface
-  cloud_firestore_web:
-    path: ../packages/cloud_firestore/cloud_firestore_web
```

Comment:
is it intended + desired to remove all these from the pubspec? I'm not sure exactly what's going on or why (apologies) I just am not used to seeing a negative line count in "tests" areas as a positive :thinking:

Reply:
We test cloud firestore in a separate app & CI job, since it's a biggie

Reply:
Firestore has its own test suite in its directory. I believe this was never removed when we ported over to a separate test app for all the other  plugins (similar to test setup for RNFB) which increases build time.

Reply:
(didn't see Mike's comment 🙈 )